### ISSUE TITLE
Dilation rate can't display correctly

### ISSUE THREAD

**Issue by su526664687** [tensorflow/tensorboard#1239]
- TensorBoard version: 1.8.0
 - TensorFlow version if different from TensorBoard: 1.9.0-dev20180522
 - OS Platform and version: Linux Ubuntu 16.04
 - Python version: 2.7
 - For browser-related issues:
   - Browser type and version: 62.0.3202.75 (Official Build) (64-bit)
   - Screenshot if it's a visual issue
![1](https://user-images.githubusercontent.com/16068384/41266755-fb360802-6e2a-11e8-967b-e2fd62190883.png)

Description: 
When I run the deeplab code in tensorflow/models (https://github.com/tensorflow/models/tree/master/research/deeplab), I found the dilation rate can't display correctly in the convolution. This dilatin rate should be [1,2,2,1], but the displayed is [1,1,1,1].
Maybe this is a bug?

**Comment by nfelt** [comment]
Thanks for the report.  Could you provide a more detailed way to reproduce this?  Ideally a tfevents file that when loaded in TensorBoard shows the unexpected dilation rate attribute.

**Comment by su526664687** [comment]
Thanks for your quick response. I give an example.

The code is below, 
```
import tensorflow as tf

X = tf.placeholder(tf.float32, [1, 10, 10, 3])
W = tf.Variable(tf.random_normal([3, 3, 3, 10]))

x = tf.nn.atrous_conv2d(X, W, rate= 2, padding= 'SAME')

with tf.Session() as sess:
    init = tf.global_variables_initializer()
    sess.run(init)
    tf.summary.FileWriter('logs/', sess.graph)
```
below is my tfevent, 
[events.out.tfevents.1529653537.sjming.tar.gz](https://github.com/tensorflow/tensorboard/files/2126733/events.out.tfevents.1529653537.sjming.tar.gz)

the graph is following, 
![1](https://user-images.githubusercontent.com/16068384/41764577-24e16a50-7634-11e8-9814-6a0ca034770e.png)

I analysis my problem, and then I found the atrous_conv is implemented by combining SpaceToBatchND, convolution and BatchToSpaceND, which transform atrous_conv to normal conv, so the dilation is always  [1,1,1,1] even if I set the rate is 2.

The conclusion is I misunderstand your tensorboard display. So sorry.
